torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -400,23 +394,17 @@ def shard_parameter(
         process_group (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used.
 
+    Returns:
+        A :class:`ShardedTensor` shared from the given tensor.
+
     .. warning::
         Only :class:`torch.distributed._sharding_spec.ShardingSpec` is
```

Comment:
This should be ChunkShardingSpec, we should fix the docs in `shard_parameter` too

Reply:
I see. So we directly assume the input is a `ChunkShardingSpec`? Don't we need to make it more generic?

Reply:
Update: Opps. Misreading the comment and will correct the error in both places.